client/my-sites/importer/importer-site-importer.jsx

```diff
@@ -23,6 +23,7 @@ class ImporterSiteImporter extends React.PureComponent {
 		// TODO: we could move this to the component itself. Here were trying to stick to a generalisation
 		// that doesn't really apply for this importer - we don't upload anything as such.
 		uploadDescription: this.props.translate( 'Type your existing site URL to start the import.' ),
+		engine: 'wix',
```

Comment:
We might want to start using sharable constants for such properties in the future, though I wouldn't worry about it doing that now.